### ISSUE TITLE
Earnings per min calculation : building cancellation

### ISSUE THREAD

**Issue by leodew** [OpenRA/OpenRA#13212]
It seems to me that building cancellation adds income to the earnings graph.

You can see here a game against an AI, that I intentionnally blocked in debug (with wire fences). Thus the AI was trying to build, then cancelling every time. The AI has a steady "income" of 2500/min, although having no source of income.

![openra-2017-04-29t182145220z](https://cloud.githubusercontent.com/assets/8159198/25557833/40616154-2d1a-11e7-93ea-953cbf55222c.png)

**Comment by rob-v** [comment]
You're right - building cancellation or selling increases player's earnings currently. 
Fixed by PR #13213.

**Comment by leodew** [comment]
Awesome @rob-v !
